assets/extensions/update-notification/wccom-connection.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Multiple from './multiple';
+import UpdateAvailableLabel from './update-available-label';
+import { Col } from '../grid';
+import { UpdateIcon } from '../../icons';
+import ExtensionActions from '../extension-actions';
+
+/**
+ * WooCommerce.com connection component.
+ *
+ * @param {Object} props            Component props.
+ * @param {Array}  props.extensions Extensions list.
+ */
+const WCCOMConnection = ( { extensions } ) => {
```

Comment:
Since this component is very similar to the `UpdateNotification`, I think if we could try a different approach to reuse the same code.
Maybe creating a common component using props in the parts they differ? Or just adding extra props to the `UpdateNotification` in order to use the same for this purpose too? WDYT?

Reply:
Since I refactored this in #4215, I'm going to leave as-is here and make any changes in that other PR.

Reply:
OK, so given recent discussions I don't think this is priority. 😅 Would you agree?

Reply:
😅 Sure! Not a big deal!